cfpb/cfgov-refresh
# Add LineChartIndex chart type

## Additions

- Adds LineChartIndex as option in Wagtail ChartBlock "Type" dropdown.

## Changes

- Adds additional text in Wagtail ChartBlock "Type" dropdown that indicates y-axis values for each chart type.
- Bumps `cfpb-chart-builder` to `5.6.0`.

## Testing

1. Run `./frontend.sh`
1. Run `python cfgov/manage.py migrate`
2. Restart local server.
3. Log into http://localhost:8000/admin/
4. Create a browse page and add a ChartBlock and required fields.
5. Set the type as "Line" and set data source as `consumer-credit-trends/auto-loans/num_data_AUT.csv` and preview to see that y-axis is millions/billions.
6. Change the type to "Line-Index" and set data source as `consumer-credit-trends/auto-loans/inq_data_AUT.csv` and preview to see that y-axis is integers and the label is "Index (January 2009 = 100)".

## Screenshots

![screen shot 2018-09-06 at 5 15 45 pm](https://user-images.githubusercontent.com/704760/45185557-8cf9ff80-b1f8-11e8-8f55-2d739ee290b5.png)

## Notes

- With the Line-Index type set, the y-axis label is hardcoded and can not be overridden from the custom y-axis field.
